code/modules/mob/living/carbon/human/ego_gifts.dm

```diff
@@ -0,0 +1,60 @@
+#define HAT "Hat_Slot"
+#define HELMET "Helmet_Slot"
+#define EYE "Eye_Slot"
+#define FACE "Face_Slot"
+#define MOUTH_1 "Mouth_Slot_1"
+#define MOUTH_2 "Mouth_Slot_2"
+#define CHEEK "Cheek_Slot"
+#define BROOCH "Brooch_Slot"
+#define NECKWEAR "Neckwear_Slot"
+#define LEFTBACK "Left_Back_Slot"
+#define RIGHTBACK "Right_Back_Slot"
+#define HAND_1 "Hand_Slot_1"
+#define HAND_2 "Hand_Slot_2"
+#define SPECIAL "Special_Other_Slot"
+/ego_gifts // Currently Covers most EGO Gift Functions, most others can be done via armors
```

Comment:
```suggestion
/datum/ego_gift // Currently Covers most EGO Gift Functions, most others can be done via armors
```
Don't forget to change it everywhere else.

Reply:
Done digity done